docs/manual-EN/5.appendix/sql-ngql.md

```diff
@@ -0,0 +1,194 @@
+# Comparison Between SQL and nGQL
+
+## Conceptual Comparisons
+
+|Name               | SQL | nGQL          |
+| --- | --- | --- |
+| vertex      | \  | vertex        |
+| edge | \    | edge          |
+| vertex type        | \   | tag           |
+| edge type          | \  | edge type     |
+| vertex identifier          | \ | vid           |
+| edge identifier        | edge id generated by default   | src, dst, rank  |
+| column | column | \ |
+| row | row | \ |
+
+## Syntax Comparisons
+
+### Data Definition Language (DDL)
+
+DDL or Data Definition Language can be used to define the database schema. It simply deals with descriptions of the database schema and is used to create and modify the structure of database objects in the database.
```

Comment:
"DDL or Data Definition Language" ==> "Data Definition Language (DDL)"

"the database schema" => "a database schema" or "database schemas"

"It simply deals with descriptions of the database schema and is used to create and modify the structure of database objects in the database." ==>redundancy should be avoided, suggestion: "DDL statements create and modify the objects of a database, such as graph vertices and edges, tables."